src/components/TableRow.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+function TableRow(props) {
+  const [isClicked, setIsClicked] = useState(true);
+  return (
+    <tr
+      onClick={() => setIsClicked(!isClicked)}
+      style={!isClicked ? { backgroundColor: "yellow" } : null}
```

Comment:
Very nice solution, well done!👏 I think it would make more sense if `isClicked` was initially set to false. And if `isClicked` is true, then the background color should change.